generateRandom.js

```diff
@@ -0,0 +1,3 @@
+const generateRandom = (from, to) => console.log(Math.floor(Math.random() * (to - from + 1)) + from);
+
+module.exports = generateRandom;
```

Comment:
each file should be closed with an empty line

Reply:
![image](https://user-images.githubusercontent.com/6584990/190259555-e1364034-0509-4b84-bed9-b6f87a7141b6.png)